packages/test/end-to-end-tests/src/test/loaderTest.spec.ts

```diff
@@ -218,4 +218,15 @@ describe("Loader.request", () => {
         assert.strictEqual(sameDataStore1, sameDataStore2,
             "same containers do not return same data store for same request");
     });
+    it("request can handle url with query params", async () => {
+        const url = await container.getAbsoluteUrl("");
+        assert(url);
```

Comment:
Can you please add a message to this assert. That will help in debugging.
I see that other tests in this file don't add message but we should do it for new tests.
nit: Also, if you would like, you can add the message to other tests in this file as well :-).